src/Modules/Processing/CustomPlayer/CustomPlayer.cpp

```diff
@@ -37,9 +38,42 @@ void CustomPlayer::exec() {
   if (!field || !frame || !robot) {
     return;
   }
+  if(!frame->has_ball()) return;
+  auto&& pos_ball = frame->ball();
+  if(field->enemyPenaltyAreaContains(pos_ball)) return;
+  if(robot->distTo(pos_ball)>150)
+  {
+    SSLMotion::GoToPoint goToPoint(pos_ball,
+                                   (pos_ball - robot->position()).angle(),
+                                   true);
+    auto command = sslNavigation.run(robot.value(), SSLRobotCommand(goToPoint));
+    command.set_dribbler(true);
+    command.set_dribblerVelocity(4);
+    emit sendCommand(command);
+  }else {
+    SSLMotion::RotateOnSelf Rotate((field->enemyGoalInsideCenter() -robot->position()).angle());
```

Comment:
Manter o padrão dos nomes dos estados
```suggestion
    SSLMotion::RotateOnSelf rotate((field->enemyGoalInsideCenter() - robot->position()).angle());
```